### ISSUE TITLE
custom host ?

### ISSUE THREAD

**Issue by darkyen** [iefserge/eshttp#4]
Can we set custom hosts ?

**Comment by iefserge** [comment]
Nope, it's not supported yet, should be pretty easy to add though.

**Comment by darkyen** [comment]
Yep, I am reading your source at the moment, any roadmaps for the library ? I'd love to contribute.

**Comment by iefserge** [comment]
Closest thing to a roadmap would be TODO section in the readme :) One big missing piece is chunked outgoing responses/requests (it already can parse incoming chunks). Or do you have something in mind?

**Comment by darkyen** [comment]
Oh not much, I just want it to be a replacement of 'http'.createServer for my use case then I can host an extremely lightweight express like server on the top of it and that lets devices talk securely for me. I wish to contribute though so wanted a starting point which is why I asked for a Roadmap.
